rest_framework-stubs/permissions.pyi

```diff
@@ -46,9 +46,9 @@ class OR(_SupportsHasPermission):
 class NOT(_SupportsHasPermission):
     def __init__(self, op1: _SupportsHasPermission) -> None: ...
 
-class BasePermissionMetaclass(OperationHolderMixin, type): ...  # type: ignore[misc]
+class BasePermissionMetaclass(OperationHolderMixin, type): ...  # type: ignore[misc,unused-ignore]
```

Comment:
This gives different mypy errors depending on Python version, hence ignoring `unused-ignore`